src/core/AutomationClip.cpp

```diff
@@ -647,8 +647,8 @@ float AutomationClip::valueAt( timeMap::const_iterator v, int offset ) const
 		float m1 = OUTTAN(v) * numValues * m_tension;
 		float m2 = INTAN(v + 1) * numValues * m_tension;
 
-		auto t2 = pow(t, 2);
-		auto t3 = pow(t, 3);
+		auto t2 = std::pow(t, 2);
```

Comment:
```suggestion
		auto t2 = t * t;
```
Same with the line below. Manually squaring and cubing is better here.